com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -294,5 +294,15 @@ public override void FinishRendering(CommandBuffer cmd)
             if (m_CreateDepthTexture)
                 cmd.ReleaseTemporaryRT(k_DepthTextureHandle.id);
         }
+
+        internal override void SwapColorBuffer(CommandBuffer cmd)
```

Comment:
Is there a reason why we do not implement it in 2D? IIRC they also have post processing

Reply:
Initially this branch was built upon the RTHandles work and they were only going to implement it for UniversalRenderer due to time constraints. Now that I am no longer built on top of their branch I suppose I can try and implement it as well for the 2D renderer? @phi-lira

Reply:
If they API is not used in 2D today to avoid risk I'd say to add in follow up PR. This is internal API so users can't access it anyway.